content/posts/gsoc-physiopy/index.md

```diff
@@ -2,6 +2,104 @@
+Emphasis is going to be given to the development breakdown, the rationale of all the contributions and the main ideas for future development.
+
+## 1. Project Overview
+{{< lead >}}
+*"Semi-Automated Workflows for Physiological Signals"*
+{{< /lead >}}
+
+In general, the project entailed the upgrade and restructuring of existing Physiopy packages, `peakdet` and `phys2denoise`, in order to be used more harmoniously with each other. 
+Following the upgrade, the development of semi-automated workflows was planned, along with the accompanying CLI. 
+
+## 2. Pull Request Synopsis
+### `peakdet`
+- https://github.com/physiopy/peakdet/pull/67
+- https://github.com/physiopy/peakdet/pull/62
+### `phys2denoise`
+- https://github.com/physiopy/phys2denoise/pull/54
+- https://github.com/physiopy/phys2denoise/pull/56
+- https://github.com/physiopy/phys2denoise/pull/57
+### `physutils`
+- https://github.com/physiopy/physutils/pull/2
+- https://github.com/physiopy/physutils/pull/4
+- https://github.com/physiopy/physutils/pull/7
+
+## 3. Development Overview
+
+### 3.1 Infrastructure 
+
+### 3.2 Optimizing peakdet/phys2denoise interoperability
+
+The `peakdet` and `phys2denoise` packages, have been both very critical packages of the physiopy suite, with their operation being frequently sequential in multiple fMRI denoising applications. 
+Meaning, that in many common data processing routines, the physiological data first needed to pass through `peakdet`, for filtering and peak/trough detection, and then through `phys2denoise` for the computation of the several denoising metrics.
+However, by nature up to this point, `peakdet` was built in an "object-oriented" fashion, being structured around the `Physio` object, while `phys2denoise` was predominantly "function-oriented".
+The main idea was to incorporate the `Physio` object into both packages, to exploit some of its powerful features, such as operation history management and history "replaying" and easy interfacing with multiple input physiological data types.
+Therefore, the main challenge was to make this migration, incorporating these features to `phys2denoise` and adding more to cater to the additional needs, while continuing to support its "function-only" use.
+
+Initially, prior to the upgrade, a new package was issued for this scope: `physutils`.
+This module would host the common physiopy suite tools, to be used throughout the physiopy packages, with one of them being the `Physio` object.
+Another honorable mention is that while `phys2denoise` has now fully migrated to use `physutils`, the community decided to proceed with such changes in a fork of the `peakdet` package, re-named as [`prep4phys`](https://github.com/physiopy/prep4phys), which will serve as the active package moving on.
+
+
+
+### 3.3 Workflow and CLI for phys2denoise
+
+For the structuring of the `phys2denoise` workflow, the `pydra` python package was used, a dataflow engine commonly used in neuroimaging. 
```

Comment:
Here I think it's good to add a few sentences on what some of the advantages of the pydra approach are (what motivated this) and to link to any tools you found helpful in developing your understanding of the package. Remember, you're trying to provide a good starting point for future developers in the community and to demonstrate the skills you fostered over the summer. What is a pydra task, for example?